Fastly-Logging-Splunk/.rpdk-config

```diff
@@ -0,0 +1,31 @@
+{
+    "artifact_type": "RESOURCE",
+    "typeName": "Fastly::Logging::Splunk",
+    "language": "typescript",
+    "runtime": "nodejs14.x",
+    "entrypoint": "dist/Fastly-Logging-Splunk/src/handlers.entrypoint",
+    "testEntrypoint": "dist/Fastly-Logging-Splunk/src/handlers.testEntrypoint",
+    "settings": {
+        "version": false,
+        "subparser_name": null,
+        "verbose": 0,
+        "force": false,
+        "type_name": null,
+        "artifact_type": null,
+        "endpoint_url": null,
+        "region": null,
+        "target_schemas": [],
+        "useDocker": false,
+        "protocolVersion": "2.0.0"
+    },
+    "contractSettings":{
+      "typeConfiguration": {
+         "configuration": {
+             "FastlyAccess": {
+                 "Token": "<FASTLY_TOKEN>"
```

Comment:
Can we replace this with the SSM resolve so that it matches most of the other providers/resources?